build-support/bin/ci.sh

```diff
@@ -58,7 +59,7 @@ python_unit_shard="0/1"
 python_contrib_shard="0/1"
 python_intg_shard="0/1"
 
-while getopts "hfxbkmsrjlpu:ny:ci:at" opt; do
+while getopts "hfxbkmsrjlpue:ny:ci:at" opt; do
```

Comment:
Ah, missed this but it explains red CI - you seperated `u` from its `:` - needs to be `u:` since `u` takes a shard argument.